wiki/Debugging.md

```diff
@@ -20,6 +20,62 @@
     while on bootsplash
 
 ### Stuck at bootanimation/splash (or need regular logs but ADB is dead)
+1) Reboot into recovery.
+2) Mount `system` in recovery.
+3) Connect to PC.
+4) Run:
+
+    ```adb shell mount -o rw,remount /system_root```
+
+    ```adb pull /system_root/system/build.prop```
+
+5) Use a text editor to change the following props in build.prop 
+
+    `ro.adb.secure=0`
+    
+    `ro.debuggable=1`
+    
+    `ro.secure=0`
+    
+    `persist.service.adb.enable=1`
+    
+    `persist.service.debuggable=1`
+    
+    `persist.sys.usb.config=adb`
+

```

Comment:
Make all of this one blockquote

Reply:
I thought it might be easier to copy them one by one this way, as all the props are scattered across the whole file

Reply:
Yeah, 6am brian like doing misfakes.

I understand your reasoning. Make them seperate but blockcodes not inline code (triple tilde)

Reply:
ok